### DISCUSSION TITLE
Does the Nuxt server side have the same assets catalog as the client side, with the packaged project assets catalog placed in the server follow directory?

### DISCUSSION THREAD

**Discussion by qcyblm** [nuxt/nuxt#28286] [Questions]
This requirement is in some projects where the front-end and back-end are not separated, and the data in the project is stored in a specified directory on the server side for adding, deleting, modifying, and querying files, such as JSON file storage and LevelDB file databases.

**Comment by Davvii1** [comment]
Are you asking if you placed a file inside `/assets` in your nuxt project, you will be able to access to the content inside from the server side and client side the same way, and when you build your project, it will remain the same?

**Reply by qcyblm** [reply]
I'm asking about putting the /assets directory in /server in a nuxt project, I can manipulate the directory through the server service, but client access can't read the directory, and when building the project the directory is placed in /.output/server/assets for readability.

Example:

```
import { readFile } from "node:fs/promises";
import { join } from "node:path";

export default defineEventHandler(
  async (event) =>
    // dev ./server/assets/data.json
    // build ./.output/server/assets/data.json
    await readFile(join(import.meta.dirname, "./assets", "./data.json")) 
);

```
